src/components/Resources/index.js

```diff
@@ -5,23 +5,44 @@ import PersonalMenu from '../PersonalMenu';
 import Search from '../Search';
 import Grid from '@material-ui/core/Grid';
 import { ResourceCard } from './ResourceCard';
+import * as helper from '../helper';
 
 function Resources({ getResourcesUrl }) {
   const [resources, setResources] = useState([]);
+  const [searchValue, setSearchValue] = useState('');
+  const [loading, setLoading] = useState(true);
```

Comment:
Let's use the React-Query library (see comment below) to follow similar patterns established in the Resource Page. This should alleviate us from needing to worry about loading and error states.

Reply:
I very much agree, but @peoray is tackling this in https://github.com/codebuddies/frontend/issues/143! How do you feel about doing the refactor in a separate PR?

Reply:
Should I mark this as resolved?

Reply:
Yes, feel free!